src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/FilterClause.java

```diff
@@ -127,4 +130,104 @@ private void validateCollectionList(
       }
     }
   }
+
+  public void invertForCollectionCommand(CommandContext<CollectionSchemaObject> commandContext) {
```

Comment:
Missing `@Overrides`

Also, need to have override for Table case, otherwise all Table-based filters fail.